src/components/Modals/FiatRamps/fiatRampProviders/banxa.ts

```diff
@@ -0,0 +1,50 @@
+import { adapters, caip19 } from '@shapeshiftoss/caip'
+import { ChainTypes } from '@shapeshiftoss/types'
+
+import { FiatRampAction, FiatRampAsset } from '../FiatRampsCommon'
+
+const banxaChainMap: Record<ChainTypes, string> = {
```

Comment:
Magic values:
- Ideally such mapping would also move to lib, like the asset ID mappings
- The naming should be made more clear what's the key and values of the mapping
- Using `chain` is not the safest discriminator, we should use `chainId` (`caip10`)

Reply:
created https://github.com/shapeshift/lib/pull/546